src/manifolds/SymmetricPositiveSemidefiniteFixedRank.jl

```diff
@@ -0,0 +1,280 @@
+@doc raw"""
+    SymmetricPositiveSemidefiniteFixedRank{n,k,𝔽} <: AbstractEmbeddedManifold{𝔽,DefaultIsometricEmbeddingType}
+
+The [`Manifold`](@ref) $ \operatorname{SPS}_k(n)$ consisting of the real- or complex-valued
+symmetric positive semidefinite matrices of size $n × n$ and rank $k$, i.e. the set
+
+````math
+\operatorname{SPS}_k(n) = \bigl\{
```

Comment:
```suggestion
\operatorname{SPS}_k(n,𝔽) = \bigl\{
```
What do you think about adding the field here?

Reply:
looks a little too crowded for a math symbol to me; I think until now we never or nearly never did that?

Reply:
It's sort of needed in formulas for manifold dimension below. Anyway, the meaning looks clear even if the field is omitted here.

Reply:
I clarified, that we might add the field here and there. We should maybe globally unify where and how we add the field to a namifold symbol in mathematical formulae.

Reply:
Thanks, great :slightly_smiling_face: . Unifying this notation makes sense.